src/Data/Connection/PostObjectConnectionResolver.php

```diff
@@ -269,20 +270,19 @@ public function get_query_args() {
 				$ids = array_map( function( $id ) {
 					return absint( $id );
 				}, $ids );
+
+				if ( ! empty( $this->args['before'] ) ) {
+					$ids = array_reverse( $ids );
```

Comment:
Passing on feedback from @hallee: The array is only reversed if a `before` cursor has been passed. This won't be the case on the first page. Seems like we additionally need to look at the `order` or comparison operator.